src/app/GitCommands/RevisionReader.cs

```diff
@@ -149,13 +149,14 @@ public IReadOnlyCollection<GitRevision> GetRevisionsFromRange(string olderCommit
         }
 
         /// <summary>
-        /// Retrieves the GitRevision.
+        ///  Retrieves the <see cref="GitRevision"/> for a real commit.
```

Comment:
OT: I do not think it is a good idea to have a convention with an extra space here if the formatter does not support it.

Reply:
It improves the readability, so +1 in my books.